data/amd/register/msr/MSR0000_0010.yml

```diff
@@ -0,0 +1,18 @@
+- name: MSR0000_0010
+  long_name: "Time Stamp Counter"
+  purpose: |
+       "
+       The TSC uses a common reference for all sockets, cores and threads. 
+       "
+  size: 64
+  arch: x86
+
+  access_mechanisms:
+      - name: rdmsr
+        is_read: True
+        address: 0x4c1
```

Comment:
Where did you find the address for this MSR? I can't figure out how to find this in the manual on page 77, is it listed somewhere else?